### DISCUSSION TITLE
Hide tests from students

### DISCUSSION THREAD

**Discussion by Hohnik** [community/community#134172] [GitHub Education]
### Hi everyone,

I'm currently creating a Github Classroom, added a python template repository with the main.py and some tests in a main_test.py. 

## Question: 
Is there a way to hide the tests from the students?

**Comment by omarmoo5** [comment]
@Hohnik  have you managed to achieve this ?

**Reply by Hohnik** [reply]
No sadly not, because the tests need to be in the same repo so the runner can find them and test against the student's code :/

**Reply by omarmoo5** [reply]
Thanks !

**Comment by Hohnik** [comment]
If you find a solution don’t hesitate to tell me please 😅
